docusaurus-docs/conda-store/references/configuration-options.md

```diff
@@ -500,34 +500,9 @@ single filename to watch.
 the number of threads on your given machine. If set will limit the
 number of concurrent celery tasks to the integer.
 
-### `conda_store_server.registry.ContainerRegistry`
+### (deprecated) `conda_store_server.registry.ContainerRegistry`
 
-`ContainerRegistry.container_registries` dictionary of registries_url
-to upload built container images with callable function to configure
-registry instance with credentials. Example configuration shown
+`ContainerRegistry.container_registries` (deprecated) dictionary of registries_url to upload built container images with callable function to configure registry instance with credentials. Example configuration shown
```

Comment:
> Example configuration shown below.

There's no example here anymore; I think the last few sentences here can just be removed.